<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="hlDialog">
       <item>
        <widget class="QLabel" name="lbNombre">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Nombre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leNombre">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhPreferUppercase</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlDialog">
       <item>
        <widget class="QLabel" name="lbCosto">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Costo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsbCosto">
         <property name="inputMethodHints">
          <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlDialog">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="lbStock">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stock</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leStock">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlDialog">
       <item>
        <widget class="QLabel" name="lbStockminimo">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stockminimo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbStockminimo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlDialog">
       <item>
        <widget class="QLabel" name="lbColor">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhPreferUppercase</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlDialog">
       <item>
        <widget class="QLabel" name="lbCategoria">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Categoria</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leCategoria">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhPreferUppercase</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btExit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btExit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
